File: hw3/facts.c


/* print square roots in C language.  R. Brown, 9/2010 */

#include <stdio.h>
#include <math.h>

double factorial(int x){
	double ans=1;
	
	for(int i=1;i<(x+1);i++){
		ans=ans*i;
	}
	return ans;
}

main()
{
  
  printf("n\tn!\n----------------\n");
  for (int n=0;  n<=5;  n++)
    printf("%d\t%f\n",n, factorial(n));
  return 0;
}

File: hw7/facts.py
# print square roots in Python language.  R. Brown, 9/2010 

from math import sqrt


def factorial(x):
	result=1
	i=1
	while (i>=1 and i<(n+1)):
		result=result*i
		i=i+1

	return result

print("n\tn!\n----------------")
n=0
while (n<6):
    print(n,"\t",factorial(n))
    n=n+1